client/src/main/java/io/pravega/client/stream/impl/StreamCutImpl.java

```diff
@@ -127,7 +130,12 @@ public ByteBuffer toBytes() {
         ByteArraySegment serialized = SERIALIZER.serialize(this);
         return ByteBuffer.wrap(serialized.array(), serialized.arrayOffset(), serialized.getLength());
     }
-    
+
+    @Override
+    public String asText() throws SerializationException {
```

Comment:
please remove it from here as its now implemented in the StreamCut interface

Reply:
done.